contracts/InterestRate/StableRate.sol

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+/**
+ * @title Logic for Venus stable rate.
+ */
+contract StableRateModel {
+    event NewStableInterestParams(
+        uint256 baseRatePerBlockForStable,
+        uint256 multiplierPerBlock,
+        uint256 multiplierPerBlockForStable,
+        uint256 jumpMultiplierPerBlockForStable,
+        uint256 kink,
+        uint256 stableRatePremium,
+        uint256 optimalStableLoanRate
+    );
+
+    /// @notice Indicator that this is an InterestRateModel contract (for inspection)
+    bool public constant isInterestRateModel = true;
+
+    uint256 private constant BASE = 1e18;
+
+    /**
+     * @notice The approximate number of blocks per year that is assumed by the interest rate model
```

Comment:
In other contracts we use /// for single line commenting variables and use multiline comments on functions